### DISCUSSION TITLE
How about creating a `<input type="datetime-local">` ?

### DISCUSSION THREAD

**Discussion by YushiroDodo63** [chakra-ui/zag#1340] [Ideas]
For the input, `type="date"` is supported by DatePicker component, but I noticed that there is no equivalent for `type="datetime-local"`.

I would be very interested to know if there are plans or discussions to introduce a datetime-local component. 
If you have an overview or details on a future implementation, please let me know. 
If that is not planned yet, but you share this idea, I am willing to contribute and even develop it myself.

**Comment by segunadebayo** [comment]
Closing as duplicate of #2858.
